indico/modules/events/surveys/fields/base.py

```diff
@@ -86,6 +86,10 @@ def wtf_field_kwargs(self):
         """Returns a dict of kwargs for this field's wtforms field"""
         return {}
 
+    def get_actual_answer(self, answer):
+        """Return the actual value of the answer for the question (necessary for field like e.g. single_choice)"""
```

Comment:
that line is way too long.

Also, isn't it more the human-friendly version?

   """Returns the human-friendly version of the answer"""